### ISSUE TITLE
future forecasting for sky map

### ISSUE THREAD

**Issue by sidt59** [Andosbor/project_number_one#33]
adding future/past dates into iframe to show forecasts or historical data

**Comment by sam-reinhard** [comment]
As is, is the date input actually doing anything? Seems like all of our data is based on current conditions so if it isn't, let's just take it out

**Comment by sidt59** [comment]
i dont think the date is doing anything...yeah i think the easiest solution is to take it out. if thats what yall wanna do then i'll close this issue out as well

**Comment by sam-reinhard** [comment]
yeah i think we should. maybe delete that and add a piece next to the zip code input that says something like "Enter your zip code to see stargazing conditions in your area"
